CMakeLists.txt

```diff
@@ -61,11 +70,35 @@ install(FILES
   src/binode.h
   DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/keyfinder")
 
-set(LIBKEYFINDER_VERSION 2.2.3)
+# pkgconfig
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/libkeyfinder.pc.cmake
   ${CMAKE_CURRENT_BINARY_DIR}/libkeyfinder.pc @ONLY)
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/libkeyfinder.pc DESTINATION "${CMAKE_INSTALL_LIBDIR}/pkgconfig")
 
+# CMake config
+set(KEYFINDER_INSTALL_CMAKEDIR "lib/cmake/KeyFinder")
+install(
+  EXPORT KeyFinderTargets
+  FILE KeyFinderTargets.cmake
+  NAMESPACE KeyFinder::
+  DESTINATION "${KEYFINDER_INSTALL_CMAKEDIR}"
+)
+configure_package_config_file(KeyFinderConfig.cmake.in
+  "${CMAKE_CURRENT_BINARY_DIR}/KeyFinderConfig.cmake"
```

Comment:
```suggestion
  "${CMAKE_CURRENT_BINARY_DIR}/keyfinderConfig.cmake"
```
?

Reply:
I think it needs to match the parameter for `find_package()`. On my system, camelcase seems to be the de facto standard:

```
$ ls /usr/lib/cmake
AccountsQt5                                           KF5Purpose                        Qt5PacketProtocol                 uwac0
AdwaitaQt                                             KF5Service                        Qt5Pdf                            WinPR2
c-ares                                                KF5Solid                          Qt5PdfWidgets                     xapian
cblas-3.9.0                                           KF5Sonnet                         Qt5PlatformCompositorSupport      ZXing
clang                                                 KF5TextWidgets                    Qt5Positioning
cmark-gfm-release.cmake                               KF5Wallet                         Qt5PositioningQuick
cmark-gfm.cmake                                       KF5WidgetsAddons                  Qt5PrintSupport
DBus1                                                 KF5WindowSystem                   Qt5Qml
...
```

Reply:
okay